crates/sui-sdk/README.md

```diff
@@ -1,6 +1,138 @@
-# Rust SDK examples
+# Sui Rust SDK
 
-This directory contains examples of interacting with a Move language smart contract using the Sui Rust SDK. See the [introduction to the Rust SDK](https://github.com/MystenLabs/sui/blob/main/doc/src/build/rust-sdk.md) for additional details.
+This crate provides the Sui Rust SDK. For the crate documentation, see [https://docs.rs/sui_sdk/](https://docs.rs/sui_sdk/)
+
+## Getting Started
+
+Add the `sui-sdk` dependency as following:
+
+```toml
+sui-sdk = { git = "https://github.com/mystenlabs/sui" }
+tokio = { version = "1.2", features = ["full"] }
+anyhow = "1.0"
+```
+
+The main building block for the Sui Rust SDK is the `SuiClientBuilder`, which provides a simple and straightforward way of connectiong to a Sui network and having access to the different available APIs. 
+
+A simple example that connects to a running Sui local network and available Sui networks is shown below. If you want to try to run this program, make sure to spin up a local network with a local validator, a fullnode, and a faucet server (see [preqrequisites](README.md) for more inforrmation).
```

Comment:
```suggestion
An example that connects to a running Sui local network and available Sui networks follows. If you want to run this program, make sure to spin up a local network with a local validator, a Full node, and a faucet server (see [preqrequisites](README.md) for more inforrmation).
```
Be careful with words like 'simple'. If it's not simple for the reader for whatever reason, you're unintentionally calling them dumb :)

Reply:
Fixed